pages/linux/rofi.md

```diff
@@ -0,0 +1,15 @@
+# rofi
+
+> An application launcher and window switcher.
+
+- Show the list of apps:
+
+`rofi -show drun`
+
+- Show the list of all commands:
+
+`rofi -show run`
+
+- Switch between windows:
+
+`rofi -show window`
```

Comment:
Is there any way to specify which one to switch to? Or is there a graphical interface that shows?

Reply:
Here:
![screenshot from 2018-01-14 05-04-17](https://user-images.githubusercontent.com/8542502/34910458-a776f576-f8e8-11e7-8f30-e9f075680061.png)

Reply:
Ah, awesome! That looks really useful. In that case, this PR looks ok to me :-)